.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,45 @@
-**Pull request recommendations:**
+# Problem
 
-- [ ] Name your pull request _your-development-type/short-description_. Ex: _feature/read-tiff-files_
-- [ ] Link to any relevant issue in the PR description. Ex: _Resolves [gh-##], adds tiff file format support_
-- [ ] Provide description and context of changes.
-- [ ] Provide relevant tests for your feature or bug fix.
-- [ ] Provide or update documentation for any feature added by your pull request.
+What is the problem this work solves, including
+[Link to story or ticket](https://my-tracking-system.url/ticket-number)
+
+# Solution
+
+What I/we did to solve this problem
+
+with @pairperson1
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- This change requires a documentation update
+- This change requires updated or new tests
+
+## Change summary:
+
+- Tidy, well formulated commit message
+- Another great commit message
+- Something else I/we did
```

Comment:
I know this is in our other templates but I don't think it's really helpful since we have a commit log. suggest deleting it.

Reply:
Feel the same way TBH-- I usually use the changes section to do the same thing. Will do!